server/server.js

```diff
@@ -1,15 +1,89 @@
 const express = require("express");
 const app = express();
-const port = process.env.PORT || 5000;
+var cors = require("cors");
+var bodyParser = require("body-parser");
+
+app.use(cors());
+app.use(bodyParser.json());
+
+const port = 5000;
 
 app.listen(port, () => console.log(`Listening on port ${port}`));
+const { Pool } = require("pg");
+
+const db = new Pool({
+  host: "localhost",
+  user: "mickeyhaile",
+  port: 5432,
+  password: "",
+  database: "cyf_videos",
+});
```

Comment:
All this information should be environment variables and shouldn't be commited to version control